src/platform-includes/sourcemaps/overview/javascript.mdx

```diff
@@ -2,13 +2,22 @@
 
 We've compiled a list of guides on how to upload source maps to Sentry for the most popular JavaScript build tools:
 
+### Sentry Plugin Support
```

Comment:
l: I think "Sentry Plugin" is very generic. Not sure if adding "bundler" is much better but I think this makes it a little clearer 🤔  

```suggestion
### Sentry Bundler Plugin Support
```